### PR TITLE
Allow tap_action: call-service for picture-entity

### PR DESCRIPTION
**Pull request by thomasloven**
Documentation: home-assistant/home-assistant.io#7427

### CODE REVIEW

File: src/panels/lovelace/cards/hui-picture-entity-card.js [hunk 1]
```diff
@@ -106,6 +106,16 @@ class HuiPictureEntityCard extends EventsMixin(LocalizeMixin(PolymerElement)) {
       throw new Error("No image source configured.");
     }
 
+    if (
+      (config.tap_action === "call-service" ||
+        config.hold_action === "call-service") &&
+      !config.service
+    ) {
+      throw new Error(
+        'Missing rewuired property "service" when tap_action or hold_action is call-service'
```
Comment (iantrich) [inline src/panels/lovelace/cards/hui-picture-entity-card.js]: `required`

**Comment by iantrich** [comment]
We should just use the common `handleClick` method instead of replicating it throughout our elements.
https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/lovelace/common/handle-click.ts

**Comment by thomasloven** [comment]
I agree, but it seems that expects an Element rather than a card?

**Comment by balloob** [comment]
Could we create a generalized function for cards?

**Comment by thomasloven** [comment]
That would be a good idea, but not in the scope of this PR.

The `handleClick` function takes a `config: LovelaceElementConfig`, but all of the properties it uses are optional, and could be added to `LovelaceConfig` as well..
I suppose typescript allows for some kind of
```interface TapActionConfig {
 hold_action?: string
tap_action?: string
... etc
}```
which you can then make both `LovelaceConfig` and `LovelaceElementConfig` extend?

I'll get on it.

**Comment by thomasloven** [comment]
Perhaps this should wait until this has been discussed: https://github.com/home-assistant/ui-schema/issues/189

**Comment by thomasloven** [comment]
Could someone with an android device could go [here](https://codepen.io/thomasloven/full/xQVwZV), click and long-press each of the squares and then tell me what happens?

**Comment by balloob** [comment]
Yes, we should put this PR on hold.

**Comment by balloob** [comment]
I also want to avoid adding features until our UI editors are a bit more mature, so we know how much we shoot ourselves in the foot with supporting new things 😉

**Pull request closed without merging**